.travis.yml

```diff
@@ -27,24 +39,28 @@ before_install:
 install:
   # Upgrade pip and setuptools. Mock has issues with the default version of
   # setuptools
-  - |
-    pip install --upgrade pip
-    pip install --upgrade setuptools
-    pip install $NUMPY
-    pip install $MPL
-    pip install -r requirements.txt
+  - pip install --upgrade pip
+  - pip install --upgrade setuptools
+
+  - pip install $NUMPY
+  - pip install $MPL
```

Comment:
There are APT binary versions for numpy 1.6.1 and matplotlib 1.1.1.  

I think I can install binary versions of freetype and libpng to speed up matplotlib compilation.

All the versions are old, but they are within minimum requirements.